src/hooks/api-queries.ts

```diff
@@ -303,15 +303,16 @@ export const useFindProjectMilestoneQuery = ({ projectId, milestoneId }: FindPro
  *
  * @param params request params
  */
-export const useListTasksQuery = ({ projectId, milestoneId, first, max }: ListTasksRequest) => {
+export const useListTasksQuery = (params: Partial<ListTasksRequest>) => {
+  const { projectId, changeProposalId, milestoneId, first, max } = params;
   const { tasksApi } = useApi();
   const { t } = useTranslation();
 
   return useQuery({
-    queryKey: ["tasks", { milestoneId, first, max }],
+    queryKey: ["projects", projectId, "tasks", { changeProposalId, milestoneId, first, max }],
```

Comment:
Is tasks still under projects? If not, this should be
`["tasks", { projectId, milestoneId, changeProposalId, first, max }]`